app/views/courses/applications-open-date.njk

```diff
@@ -30,9 +30,14 @@
 
       {% set headingHtml %}
         {% include "_includes/page-heading-legend.njk" %}
-      {% endset %}
 
-      <form action="{{ actions.save }}" method="post" accept-charset="utf-8" novalidate>
+    <form action="{{ actions.save }}" method="post" accept-charset="utf-8" novalidate>
+
+    <p class="govuk-body">We recommend that you open the course for applications as soon as the course is on Find to get as many applications as possible.</p>
```

Comment:
I don't know why, but the content looks a little squished under the header. Probably not a train smash as this is the prototype.